lib/ide/src/controller/module.rs

```diff
@@ -68,8 +123,8 @@ shared! { Handle
     pub struct Controller {
         /// This module's location.
         location: Location,
-        /// The current module ast, used by synchronizing both module representations.
-        ast: Ast,
+        /// The current module used by synchronizing both module representations.
```

Comment:
I dont understand this documentation at all. What is a module used by synchronizing both module representations? I read it like 10 times and this doesnt make any sense to me.

Reply:
I don't either, it was Adam comment AFAIK.

Reply:
@Josef-Vonasek please write to Adam on Discord and try to learn it and fix that if Adam replies today. If not, then he will fix that